statistics/row_sampler.go

```diff
@@ -129,6 +147,7 @@ func NewReservoirRowSampleCollector(maxSampleSize int, totalLen int) *ReservoirR
 		FMSketches: make([]*FMSketch, 0, totalLen),
 		TotalSizes: make([]int64, totalLen),
 	}
+	base.MemSize = int64(unsafe.Sizeof(base.Samples)) + int64(unsafe.Sizeof(base.NullCount)) + int64(unsafe.Sizeof(base.FMSketches)) + int64(unsafe.Sizeof(base.TotalSizes)) + 4
```

Comment:
Do we want to calculate the capacity of those slices or just the memory of `baseCollector` struct. The current calculation seems to be the latter and `unsafe.Sizeof(base)` may be enough. Besides, if the init memory size is small enough, maybe we don't need to consume it before adding items into it.

Reply:
remove it.